zebra/explorer.md

```diff
@@ -13,23 +13,23 @@ https://zcashblockexplorer.com/
+| getaddressdeltas            | NO.                | We need to implement almost fully.<br>This might need a new index in the database.                                                                                                                                                                                  | [Zcash](https://zcash.github.io/rpc/getaddressdeltas.html)                                                                                                                   | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/address_controller.ex#L29) - [2](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/address_controller.ex#L73)                                                                                                                                                                                                                                                                                                                                                                     |
+| z_listunifiedreceivers      | MINING.                | Not implemented but I'll implement as part of the mining project: https://github.com/ZcashFoundation/zebra/issues/6030                                                                                         | [Zcash](https://zcash.github.io/rpc/z_listunifiedreceivers.html)                                                                                                             | [1](https://github.com/nighthawk-apps/zcash-explorer/blob/main/lib/zcash_explorer_web/controllers/address_controller.ex#L96)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Changed to "MINING" so we know it will be done before we start the explorer work.